src/Screens/Main/Chat/Input.tsx

```diff
@@ -38,28 +44,46 @@ export default ({ buddyId }: Props) => {
     sendMessage({ buddyId, text: messageContent });
   };
 
+  const inputContainerBg = showPending ? colors.lightestGray : colors.white;
+
+  React.useEffect(() => {
+    if (isPending) {
+      let pendingTimeOut = setTimeout(() => setShowPending(true), 500);
+
+      return () => clearTimeout(pendingTimeOut);
+    } else {
+      setShowPending(false);
+    }
+  }, [isPending]);
+
   return (
     <SafeAreaView style={styles.container} forceInset={{ bottom: 'always' }}>
-      <RN.View style={styles.inputContainer}>
+      <RN.View
+        style={[styles.inputContainer, { backgroundColor: inputContainerBg }]}
+      >
         <RN.TextInput
-          style={styles.inputText}
+          style={[styles.inputText]}
           onChangeText={storeMessage}
           value={messageContent}
           multiline={true}
-          editable={true}
+          editable={isPending ? false : true}
```

Comment:
this excess. !Pending : D?

Reply:
easier to understand the negation would be IMO.

Reply:
lol yeah, brainfart i guess =)